### DISCUSSION TITLE
How do i use my own network proxy with pre existing options

### DISCUSSION THREAD

**Discussion by TechGigant** [copy/v86#847] [Q&A]
i cant seem to find a way on how to do it

**Comment by Darin755** [comment]
Can you provide more details? Are you talking about the library or the v86 website?

**Reply by TechGigant** [reply]
The v86 website

**Reply by Darin755** [reply]
You just change the url

**Reply by TechGigant** [reply]
how

**Reply by spetterman66** [reply]
host your own copy of websockproxy and specify it in the emulator in a custom profile as for example wss://127.0.0.1 (use external ip if accessed from somewhere else)

**Reply by Darin755** [reply]
1.1.1.1 is the cloudflare DNS service. Did you mean 127.0.0.1?

**Reply by TechGigant** [reply]
Can you download the existing images on the v86 website and if you can how?
